unlock-app/src/components/content/event/Layout/PastEvent/LockExplorerLink.tsx

```diff
@@ -0,0 +1,35 @@
+import { PaywallLockConfigType } from '@unlock-protocol/core'
+import { Placeholder } from '@unlock-protocol/ui'
+import { ExplorerLink } from '~/components/interface/AddressLink'
+import { useMetadata } from '~/hooks/metadata'
+
+interface LockExplorerLinkProps {
+  lockAddress: string
+  network: number
+  lockCheckoutConfig: PaywallLockConfigType
+}
+export const LockExplorerLink = ({
```

Comment:
Instead of making it specific by contract let's have this handle the full `checkoutConfig` and then have it handle the logic.
Also. I think it would be better to show contract _inline_: `Contracts: Ticket1, Ticket2, Ticket3`

Reply:
Done!